pages/integrations/sso/google_workspace_saml.md.erb

```diff
@@ -8,6 +8,10 @@ To complete this tutorial, you will need admin privileges for both Google Worksp
   <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
+As a result of this tutorial, you will enable users to sign into Buildkite by clicking the Buildkite icon in the Google Apps menu. You can find this menu by clicking the 9-dot "waffle" icon.  
```

Comment:
Let's be more direct about this

> After following these steps, your Google Workspace users will be able to sign into Buildkite using their google account. 

the Google Apps menu is incidental, let's move that to end of the tutorial. Do you agree @karensawrey ?

Reply:
I do agree! Your suggestion of "here is a screenshot of what you should be seeing now" trumps the current logic of "here's a screenshot for you to know what you're trying to accomplish".